app/components/settings/profile-page/username-section.ts

```diff
@@ -9,7 +13,18 @@ interface Signature {
   };
 }
 
-export default class UsernameSectionComponent extends Component<Signature> {}
+export default class UsernameSectionComponent extends Component<Signature> {
+  @service declare authenticator: AuthenticatorService;
+
+  @tracked isSyncingGitHubUsername = false;
+
+  @action
+  async refreshFromGitHub() {
+    this.isSyncingGitHubUsername = true;
+    await this.authenticator.currentUser?.syncGitHubUsername(null);
```

Comment:
do you think it's better to keep the 'refresh from github' name consistent everywhere

Reply:
`sync` is fine, we use it across many `core` actions so it's consistent with those. The actual text for the button might change over time (based on what makes most sense to a user), so not super important to keep that in sync with everything else. 

Would recommend renaming to `syncUsernameFromGitHub` though, to avoid confusion around whether we're actually updating the username, or only `user#githubUsername`.